src/CortexPE/task/ElytraRocketBoostTrackingTask.php

```diff
@@ -65,7 +64,7 @@ public function onRun(int $currentTick){
 			)));
 			$this->internalCount++;
 		}else{
-		    $plug = PMServer::getInstance()->getPluginManager()->getPlugin(Main::getInstance()->getName());
+		    $plug = Main::getInstance();
 			$plug->getScheduler()->cancelTask($this->getTaskId());
```

Comment:
```
$plug = Main::getInstance();
$plug->getScheduler()->cancelTask($this->getTaskId());
```
could've been easily
`Main::getInstance()->getScheduler()->cancelTask($this->getTaskId());`
since you're only using the variable's value once. xd

Reply:
xD ok i'm changing that